app/src/androidTest/java/com/github/fribourgsdp/radio/AddPlaylistActivityTest.kt

```diff
@@ -104,7 +120,6 @@ class AddPlaylistActivityTest {
     }
```

Comment:
I think you forgot to remove line 119 to release Intents as you do a _@After_ ;)
That might be the reason why tests are failing...